pkg/operation/botanist/controlplane/clusterautoscaler/bootstrap.go

```diff
@@ -33,12 +34,25 @@ const (
 	managedResourceControlName = "cluster-autoscaler"
 )
 
-// TimeoutWaitForManagedResource is the timeout used while waiting for the ManagedResources to become healthy
-// or deleted.
-var TimeoutWaitForManagedResource = 2 * time.Minute
+// Bootstrapper contains functions for bootstrapping a system for the cluster-autoscaler.
+type Bootstrapper interface {
+	component.DeployWaiter
+}
```

Comment:
Can you elaborate why we need a `Bootstrapper` interface instead of just working with `component.DeployWaiter`?

Reply:
No reason, I remove it, thanks

Reply:
Can you check again, it's still showing the interface in the PR?